README.md

```diff
@@ -101,6 +101,9 @@ fn sum_of_squares(input: &[i32]) -> i32 {
 }
 ```
 
+Note that the `sum()` call is not a sequential one. It is, precisely,
+a logarithmic, tree-like reduction known from, for example, OpenMP.
```

Comment:
I appreciate what you're trying to say, but it's worded a bit awkwardly, especially with two parentheticals.  Can you try simplifying the second sentence?

Reply:
I completely forgot about it. Is the change still relevant or should I go ahead simplifying it?

Reply:
There's a new paragraph following the example you're describing, but I'm not sure if that leads the reader in the same direction that you're addressing.  I'll let you look at the context to see if you think your addition is still needed.  (My own familiarity makes it harder to judge what could be confusing for newcomers...)